bt2_values_generator.rb

```diff
@@ -81,19 +94,51 @@ class BTValueCLass::Bool < BTValueCLass::Scalar
     @bt_type_is = 'bt_value_is_bool'
     @bt_return_type = 'bt_bool'
     @bt_default_value = 'BT_FALSE'
+
+    def initialize(name,default_value)
+      bt_type = self.class.instance_variable_get(:@bt_type)
+      if !default_value.nil?
+        unless [true, false].include? default_value
+          raise "Bad default_value for '#{name}' in params.yaml, it must be #{bt_type}" 
+        end
+        default_value = { true => 'BT_TRUE', false => 'BT_FALSE' }[default_value]
```

Comment:
Let just allow `BT_TRUE` and `BT_FALSE` in the yaml, no true/ false

Reply:
What do you think if we allow true/false, since in the command line interface we allow them, then it more intuitive for the user ....